### DISCUSSION TITLE
Client Authentication is not happening using apache reverse proxy server

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#345] [General]
Dear All, I have deployed apache reverse proxy server in front of my docker EJBCA CE server and trying to do client authentication but I am getting AHO2268: Proxy client certificate callback, downstream server wanted client certificate but none are configured.

I am not presenting the client certificate from proxy server because in my scenario I want my client/user to present the RA certificate rather proxy server. 

My Scenario is: 

User client browser <-----------> Apache Reverse Proxy Server (DMZ) <------------> EJBCA SERVER (docker Container)

Here I want to confirm that I my reverse proxy server is a separate machine and I cannot make my EJBCA server to have reverse proxy because it is not in DMZ.

Any help shall highly be appreciated. Secondly the below given link was not helpful for me.

 https://github.com/Keyfactor/keyfactorcommunity/tree/main/deployment-examples/docker-engine/ejbca-ce-three-level-architecture

**Comment by merryo11** [comment]
Please also see my virtual host setting with proxy setting
![Apache-SSL-Proxy-Setting](https://github.com/Keyfactor/ejbca-ce/assets/144104956/477657d1-b0c8-4ecd-aa56-214f4cc73eb7)

**Comment by primetomas** [comment]
EJBCA container is designed to be easily used behind an ingress.
See here: https://hub.docker.com/r/keyfactor/ejbca-ce, section Using the container behind a proxy
